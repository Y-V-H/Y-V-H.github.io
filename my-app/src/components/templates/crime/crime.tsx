import React from "react";
import { Header } from '../../atoms/header/header'

export const Crime = () => {
    return (
        <Header size='h1' children="Crime page"/>
    )
}