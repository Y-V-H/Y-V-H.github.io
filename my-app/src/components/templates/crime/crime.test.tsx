import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom'
import { Crime } from './crime';

test('should render Crime template', () => {
    const component = renderer.create(
        <Crime />
    );
    expect(component.toJSON()).toMatchSnapshot();
});