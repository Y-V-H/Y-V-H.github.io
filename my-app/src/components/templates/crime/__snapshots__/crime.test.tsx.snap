// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Crime template 1`] = `
<h1
  className="a-header a-header--h1 undefined"
>
  Crime page
</h1>
`;
